src/encoded/static/components/search.js

```diff
@@ -700,12 +705,94 @@ TextFilter.propTypes = {
 // Displays the entire list of facets. It contains a number of <Facet> components.
 export const FacetList = (props) => {
     const { context, facets, filters, mode, orientation, hideTextFilter, addClasses, docTypeTitleSuffix, supressTitle, onFilter } = props;
+
+    const [expandedFacets, setExpandFacets] = React.useState(new Set());
```

Comment:
It seemed as if passing () => {} makes no difference.

Lines 275 and 329 of
https://github.com/facebook/react/blob/df14b5bcc163516fc0f1ad35e9b93732c66c1085/packages/react-dom/src/server/ReactPartialRendererHooks.js#L275

https://stackoverflow.com/questions/55621212/is-it-possible-to-react-usestate-in-react